doc/Language/grammar_tutorial.pod6

```diff
@@ -70,25 +104,48 @@ When a method (regex, token or rule) matches in the grammar, that matched string
     =end code
 
-So in this, if you were to my $match = My::Gram.parse( $string ) -- and your string started with the characters 'clever_text_keyword', you would get a match object back that contained 'clever_text_keyword' keyed by the name of 'thingy' in your match object. These can grow much more complex, as your needs require, as you might imagine.
+So in this, if you were to my $match = My::Gram.parse( $string ) -- and your
+string started with the characters 'clever_text_keyword', you would get a match
+object back that contained 'clever_text_keyword' keyed by the name of 'thingy'
+in your match object. These can grow much more complex, as your needs require,
+as you might imagine.
 
-Now, to mention TOP. The TOP method (regex, token or rule) is the overarching regex that must match everything (by default). If the string you pass in to parse doesn't match the TOP regex, your returned match object will be empty (Any).
+Now, to mention TOP. The TOP method (regex, token or rule) is the overarching
+regex that must match everything (by default). If the string you pass in to
+parse doesn't match the TOP regex, your returned match object will be empty
+(Any).
 
-As you can see above, in TOP, the "<thingy>" token is mentioned. The <thingy> is defined on the next line, "token thingy...". That means that 'clever_text_keyword' B<must> be the first thing in the string passed in, or the grammar parse will fail, and we'll get an empty match. This is great for recognizing malformed stuff that someone might give you that should be thrown away.
+As you can see above, in TOP, the "<thingy>" token is mentioned. The <thingy>
+is defined on the next line, "token thingy...". That means that
+'clever_text_keyword' B<must> be the first thing in the string passed in, or the
+grammar parse will fail, and we'll get an empty match. This is great for
+recognizing malformed stuff that someone might give you that should be thrown
+away.
 
 =head1 Learning By Example - a REST Contrivance
 
-Let's suppose we'd like to parse a URL into the component parts that make up a RESTful request. Let's decide that we want the URLs to work like this:
+Let's suppose we'd like to parse a URL into the component parts that make up a
+RESTful request. Let's decide that we want the URLs to work like this:
 
 
-    =item1 The first part of the URI we'll call the "subject", like a part, or a product, or a person.
-    =item1 The second part of the URI we'll call the "command", like standard CRUD stuff (create, retrieve, update, or delete).
-    =item1 The third part of the URI will be arbitrary data. Perhaps the specific ID we'll be working with, or a long list of data separated by "/"'s.
-    =item1 When we get a URL, we'll want 1-3 above to be placed into a nice data structure we can use without having to do all sorts of splitting, and that can be easily altered in the future or expanded upon (or extended).
+    =item1 The first part of the URI we'll call the "subject", like a part, or
```

Comment:
How about indentation that lines up like this:
item somehting, somehting
        something else